flow/src/types.rs

```diff
@@ -21,20 +21,18 @@ impl Default for FlowProtocol {
 #[derive(Encode, Decode, Clone, PartialEq, Eq, PartialOrd, Ord, TypeInfo, MaxEncodedLen)]
 #[cfg_attr(feature = "std", derive(Debug))]
 #[repr(u8)]
-pub enum FlowState {
-	Init = 0,
-	Active = 1,
-	Paused = 2,
-	Finalizing = 3,
-	Reverting = 4,
-	Success = 5,
-	Failed = 6,
-	Locked = 7,
+pub enum CampaignState {
+	Created = 0,	// waiting for start block
```

Comment:
state vs acted on, state == active but some other entity activated, so in general that is why you would use e.g. state == 'active' etc

Reply:
The only reason was having similar names for the states or at least similar approaches.
This is what Signal states look like in the main branch right now:
```
pub enum ProposalState {
	Init = 0,      // waiting for start block
	Active = 1,    // voting is active
	Accepted = 2,  // voters did approve
	Rejected = 3,  // voters did not approve
	Expired = 4,   // ended without votes
	Aborted = 5,   // sudo abort
	Finalized = 6, // accepted withdrawal proposal is processed
}
```
`Active`, but all of the others are in the Past tense.

I would be happy to rename it if you suggest the approach. I'm not sure that I got your point. Should I rename only `Activated` -> `Active` or everything else?
BTW `Activated` means it was activated when the start block number was reached.

Reply:
no, ofc it is about all the states. and yes, it is clear, what the purpose of active is.

Reply:
@2075 please, fill in the question mark placeholders, and I'll populate the changes.

Campaign states:
- Created -> **?**
- Activated -> **Active**
- Paused -> **?**
- Succeeded -> **?**
- Failed -> **?**
- Locked -> **?**

Proposal states:
- Created -> **?**
- Activated -> **Active**
- Accepted -> **?**
- Rejected -> **?**
- Expired -> **?**
- Aborted -> **?**
- Finalized -> **?**

Reply:
this is not about following blind suggestions but for discussion, I also do not have a better shot yet :)